osu.Server.Queues.ScoreStatisticsProcessor/Commands/Maintenance/VerifyImportedScoresCommand.cs

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Dapper;
+using McMaster.Extensions.CommandLineUtils;
+using osu.Server.QueueProcessor;
+using osu.Server.Queues.ScoreStatisticsProcessor.Commands.Queue;
+using osu.Server.Queues.ScoreStatisticsProcessor.Helpers;
+
+namespace osu.Server.Queues.ScoreStatisticsProcessor.Commands.Maintenance
+{
+    [Command("verify-imported-scores", Description = "Verifies data and pp for already imported scores")]
+    public class VerifyImportedScoresCommand
+    {
+        /// <summary>
+        /// The high score ID to start deleting imported high scores from.
```

Comment:
```suggestion
        /// The high score ID to start verifying imported high scores from.
```

i sure hope the current version is incorrect :sweat_smile: :sweat_smile: :sweat_smile: